integration-tests/gatsby-cli/__tests__/repl.js

```diff
@@ -0,0 +1,51 @@
+import { GatsbyCLI } from "../test-helpers"
+
+const timeout = seconds =>
+  new Promise(resolve => {
+    setTimeout(resolve, seconds * 1000)
+  })
+
+const MAX_TIMEOUT = 2147483647
+jest.setTimeout(MAX_TIMEOUT)
+
+describe(`gatsby repl`, () => {
+  const cwd = `gatsby-sites/gatsby-repl`
+
+  beforeAll(() => GatsbyCLI.from(cwd).invoke(`clean`))
+  afterAll(() => GatsbyCLI.from(cwd).invoke(`clean`))
+
+  it(`starts a gatsby site on port 8000`, async () => {
+    // 1. Start the `gatsby develop` command
+    const [childProcess, getLogs] = GatsbyCLI.from(cwd).invokeAsync(`repl`)
+
+    // 2. Wait for the build process to finish
+    await timeout(4)
```

Comment:
Might fail because of that on slow CIs.

Reply:
Yeah i realize that. I was just going to do the skateboard implementation and improve it later if it needed more stability. So far it's seemed to have worked.